src/battle-tooltips.ts

```diff
@@ -1512,7 +1515,7 @@ class BattleTooltips {
 		if (move.category !== 'Status' && !noTypeOverride.includes(move.id)) {
 			if (move.type === 'Normal') {
 				value.abilityModify(this.battle.gen > 6 ? 1.2 : 1.3, "Aerilate");
-				value.abilityModify(this.battle.gen > 6 ? 1.2 : 1.3, "Galvanize");
+				value.abilityModify(1.2, "Galvanize");
```

Comment:
I guess Galvanize didn't exist in Gen 6?

Reply:
Honestly, why not match the others? Maybe it'll come in handy for OMs that want to backport abilities?

Reply:
I guess